runtime/doc/api.txt

```diff
@@ -408,13 +408,13 @@ id. Thus, instead of >lua
 <
 use >lua
   -- create the highlight through an extmark
-  extid = vim.api.nvim_buf_set_extmark(buf, ns_id, hl_group, line, col_start, {end_col = col_end, hl_group = hl_group})
+  extid = vim.api.nvim_buf_set_extmark(buf, ns_id, line, col_start, {end_col = col_end, hl_group = hl_group})
 
   -- example: modify the extmark's highlight group
   vim.api.nvim_buf_set_extmark(buf, ns_id, NEW_HL_GROUP, line, col_start, {end_col = col_end, hl_group = hl_group, id = extid})
```

Comment:
Wait this one is also wrong😅

Reply:
You're just artificially inflating your commit stats, aren't you :)

Reply:
No I just glanced over #23310 and the line passing `NEW_HL_GROUP` to `nvim_buf_set_extmark()` was obviously wrong to me. The one with `NEW_LINE` stood out less...